app/src/main/res/navigation/nav_graph.xml

```diff
@@ -0,0 +1,180 @@
+            app:destination="@+id/bottomMenuDialog" />
+    </fragment>
+
+    <fragment
+        android:id="@+id/settingsListFragment"
+        android:name="com.saltedge.authenticator.features.settings.list.SettingsListFragment"
+        android:label="SettingsListFragment">
+
+        <action
+            android:id="@+id/about_list"
+            app:destination="@+id/aboutListFragment" />
+
+        <action
+            android:id="@+id/passcode_edit"
+            app:destination="@+id/passcodeEditFragment" />
+
+        <action
+            android:id="@+id/language"
+            app:destination="@+id/languageSelectDialog" />
+    </fragment>
+
+    <fragment
+        android:id="@+id/passcodeEditFragment"
+        android:name="com.saltedge.authenticator.features.settings.passcode.PasscodeEditFragment"
+        android:label="PasscodeEditFragment">
+
+        <action
+            android:id="@+id/settings_list"
+            app:destination="@+id/settingsListFragment" />
+    </fragment>
+
+    <dialog
+        android:id="@+id/languageSelectDialog"
+        android:name="com.saltedge.authenticator.features.settings.language.LanguageSelectDialog"
+        android:label="LanguageSelectDialog" />
+
+    <fragment
+        android:id="@+id/aboutListFragment"
+        android:name="com.saltedge.authenticator.features.settings.about.AboutListFragment"
+        android:label="AboutListFragment">
+
+        <action
+            android:id="@+id/license"
+            app:destination="@+id/licensesFragment" />
+
+        <action
+            android:id="@+id/terms_of_services"
+            app:destination="@+id/webViewFragment" />
+    </fragment>
+
+    <fragment
+        android:id="@+id/licensesFragment"
+        android:name="com.saltedge.authenticator.features.settings.licenses.LicensesFragment"
+        android:label="LicensesFragment">
+
+        <action
+            android:id="@+id/terms_of_services"
+            app:destination="@+id/webViewFragment" />
+    </fragment>
+
+    <fragment
+        android:id="@+id/webViewFragment"
+        android:name="com.saltedge.authenticator.widget.fragment.WebViewFragment"
+        android:label="WebViewFragment" />
+
+
+    <fragment
+        android:id="@+id/connectionsListFragment"
+        android:name="com.saltedge.authenticator.features.connections.list.ConnectionsListFragment"
+        android:label="ConnectionsListFragment">
+
+        <action
+            android:id="@+id/consents_list"
+            app:destination="@+id/consentsListFragment" />
+
+        <action
+            android:id="@+id/edit_connection_name_dialog"
+            app:destination="@+id/editConnectionNameDialog" />
+
+        <action
+            android:id="@+id/delete_connection_dialog"
+            app:destination="@+id/deleteConnectionDialog" />
+
+        <action
+            android:id="@+id/connect_provider"
+            app:destination="@+id/connectProviderFragment" />
+    </fragment>
+    <fragment
```

Comment:
add new line before

Reply:
OK I'll apply it everywhere in the file

Reply:
good approach. i will not show all repeating issues, just one time. check all places where the issue can appear.